scripts/generate_sitemap.py

```diff
@@ -160,10 +187,14 @@ def generate(self):
         # TODO: server side cursor query wrapper might be useful as the index gets larger.
         # Sitemap indexable limit check
         if self.sitemap_count > settings.SITEMAP_INDEX_MAX * .90: # 10% of urls remaining
-            logger.exception('WARNING: Max sitemaps nearly reached: {} of 50000 max sitemaps for index file'.format(str(self.sitemap_count)))
-            log_exception()
+            sentry.log_message('WARNING: Max sitemaps nearly reached: {} of 50000 max sitemaps for index file'.format(str(self.sitemap_count)))
```

Comment:
Ensure that the title for this one is the same from error to error as in the message below.